packages/scoped-custom-element-registry/src/scoped-custom-element-registry.ts

```diff
@@ -592,17 +593,19 @@ if (!ShadowRoot.prototype.createElement) {
     // Emulate the native RadioNodeList object
     const RadioNodeList = (class
       extends Array<Node>
-      implements Omit<RadioNodeList, 'forEach' | 'item'> {
+      implements Omit<RadioNodeList, 'forEach'> {
```

Comment:
why not implement 'forEach' as well?

Reply:
I couldn't get the types right :/ The implementation is already there in Array though.